controllers/operands/kubevirt.go

```diff
@@ -294,7 +301,73 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+
+	qps, ok := configMap.Data[qps]
+
+	if !ok {
+		return nil, fmt.Errorf("queryPerSeconds parameter not found in configmap " + configMap.Name)
+	}
+
+	qpsConverted, err := strconv.ParseFloat(qps, 32)
+
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert queryPerSeconds parameter to float; %w", err)
+	}
+
+	burst, ok := configMap.Data[burst]
```

Comment:
`brust` is already defined in this package as a constant. Please rename.

Reply:
Done.